### DISCUSSION TITLE
Virtual Desktop with Steam Big Picture

### DISCUSSION THREAD

**Discussion by hdezroberto00** [ClassicOldSong/Apollo#98] [Q&A] [answered]
So i've recently installed Apollo and everything works great. The built in Virtual Display is awesome, so i would like to launch a virtual display with Steam Big Picture. In Apollo you can't change the configuration of Virtual Desktop so i can't add the Steam command. Anyone have an idea of how can I start Steam BP directly in the Virtual Display?

**Comment by ClassicOldSong** [comment]
Go to web ui, edit `Steam` app, check `Always use Virtual Display`

**Reply by hdezroberto00** [reply]
It kinda worked (after connecting, setting virtual screen as main monitor, disconnecting and connecting again), but some games still boot up in the physical monitor so i have to enter with Anydesk and change the monitor. It would be a great adition if you put the "do-command" and "undo-command" for the physical monitor to turn off when connecting to a virtual display (using MultiMonitorTool or something like that i guess). Besides that, you have an awesome app, I recently switched from Sunshine and it's a huge upgrade, I'm really expecting to see the iOS app, so I'll keep and eye on here. Great work!

**Reply by ClassicOldSong** [reply, marked as answer]
You can set to "Only display on 2". The next time you stream it'll automatically turn off your physical monitor.

If you just need to move windows between monitors, press Win + Shift + Left/Right on the active window.

**Reply by hdezroberto00** [reply]
In which part of Apollo do I have to set the "Only display on 2"?

**Reply by ClassicOldSong** [reply]
Set it in Windows display settings. Apollo don't manage display config/conbination actively.

**Reply by hdezroberto00** [reply]
Oh wow! Got it, it works flawlessly, amazing. Keep up with the great work, i love this.
